MoreLinq/Extensions.g.cs

```diff
@@ -4154,6 +4154,7 @@ public static partial class PartitionExtension
         /// A tuple of elements satisfying the predicate and those that do not,
         /// respectively.
         /// </returns>
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null</exception>
```

Comment:
Use `<see langword="null"/>` instead of `null` consistently for new updates to docs?

Reply:
Good catch!